lib/elixir_boilerplate_graphql/types/application.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ElixirBoilerplateGraphQL.Types.Application do
```

Comment:
To be consistent with the `<Domain>.<Context>.<Module>` pattern, this should be named:

```suggestion
defmodule ElixirBoilerplateGraphQL.Application.Types do
```